spec_11/index.js

```diff
@@ -0,0 +1,13 @@
+
+const findFactorial = ( n ) => {
+  let factorial = 1
+  for ( let i = 1; i <= n; i++ ) {
```

Comment:
I completely agree, even though its been a long time, I feel essay format is so entrenched in my brain it just makes sense to use that as a way to organize code as well.